inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -71,6 +71,40 @@ std::string get_special_opset_for_op(const ngraph::Node::type_info_t& type_info)
     return "";
 }
 
+namespace rt_info {
+const std::vector<std::string> list_of_names {
```

Comment:
`static const`?

Reply:
I don't see reason to add `static`. Variable hides in unnamed namespace.